src/main/java/io/jenkins/plugins/bitbucketpushandpullrequest/environment/BitBucketPPREnvironmentContributor.java

```diff
@@ -91,6 +91,10 @@ private void setEnvVarsForCloudPullRequest(EnvVars envVars, BitBucketPPRPullRequ
     String pullRequestId = cause.getPullRequestPayLoad().getPullRequestId();
     putEnvVar(envVars, "BITBUCKET_PULL_REQUEST_ID", pullRequestId);
     LOGGER.log(Level.FINEST, "Injecting BITBUCKET_PULL_REQUEST_ID: {0}", pullRequestId);
+
+    String actor = cause.getPullRequestPayLoad().getDisplayName();
```

Comment:
@marcosmko are you sure `getDisplayName` returns anything other than null? Correct me if I am wrong: If you check the class `hudson.model.InvisibleAction` the method is not overriden / is final.

```java
    public final String getDisplayName() {
        return null;
    }
```
Did you mean

```java
String actor = cause.getRepositoryPayLoad().getPayload().getActor().getDisplayName();
```

?
(And yes, the method name is confusing because `getPulllRequestPayload` actually returns a `BitBucketPPRRepositoryAction`... 🙈)

Reply:
yeah you're right!
I called the wrong function, it shoud be getUser() instead. Fixed it, thanks!